tests/cases/fourslash/moveToFile_namespaceImport5.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /a.ts
+//// export type ExecutionPoint = string;
+////
+// @Filename: /b.ts
+//// export type ExecutionPoint = { point: string };
+////
+// @Filename: /c.ts
+//// import * as A from "./a";
+////
+//// [|async function fn1(point: A.ExecutionPoint) {}|]
+////
+// @Filename: /point.ts
+//// import * as A from "./c";
+////
+//// async function fn2(point: A.ExecutionPoint) {}
+
+verify.moveToFile({
+    newFileContents: {
+        "/c.ts": `
+`,
+        "/point.ts":
+`import * as A from "./a";
+import * as A from "./c";
```

Comment:
As we can see here, those refactorings introduce issues into target files. This is not a new problem from what I can tell. This refactoring just never tried to rename inserted symbols to avoid name clashes etc. 

At first, I thought those would be test cases that should be covered by this PR so I added them. I could just remove them now but I think they still have some value here, especially if this naming conflict is ever meant to be addressed in the future.